packages/liveblocks-yjs/src/index.ts

```diff
@@ -133,10 +161,19 @@ export class LiveblocksYjsProvider<
     }
   };
 
-  private updateDoc = (update: string, guid?: string) => {
+  private getUniqueUpdateId = (update: Uint8Array) => {
+    const clock =
+      Y.parseUpdateMeta(update).to.get(this.rootDoc.clientID) ?? "-1";
+    return this.rootDoc.clientID + ":" + clock;
+  };
```

Comment:
Is the idea behind this to extract the `clockId` behind the update and appending it to the Y.Doc's client ID? Not suggesting but just curious if the appending to the client ID necessary?

Reply:
yes, it's just a way to identify the update. Otherwise we would need to generate a UUID or nanoid with each update and return it from the server.